src/bootstrap/bootstrap.py

```diff
@@ -79,6 +79,8 @@ def get(base, url, path, checksums, verbose=False):
                 eprint("removing", temp_path)
             os.unlink(temp_path)
 
+def curl_version():
+    return float(re.match(bytes("^curl ([0-9]+\\.[0-9]+)", "utf8"), require(["curl", "-V"]))[1])
```

Comment:
Let's make this a bit more robust; if `re.match` returns `None`, then return e.g. `(1, 0)` or `(0, 0)` from the function, instead of crashing.

It's better to compare versions using tuples of integers rather than floats, so let's return `(major, minor)`.